pyqtgraph/graphicsItems/ScatterPlotItem.py

```diff
@@ -835,11 +835,14 @@ def paint(self, p, *args):
             else:
                 # render each symbol individually
                 p.setRenderHint(p.Antialiasing, aa)
+                
+                vb = self.getViewBox()
+                offset = vb.parentItem().pos() if vb.parentItem() else vb.pos()
```

Comment:
Could you describe a little more about what you're trying to achieve? This `offset` variable doesn't appear to be used anywhere...